### ISSUE TITLE
RaidSortOptions sort overwritten by addon defaults, re-applied every few seconds

### ISSUE THREAD

**Issue by buzzblizard** [Krealle/Cell-Snippets#3]
My raid frame sort order swaps between RaidSortOptions and the Cell addon defaults every 3 seconds or so whilst out of combat.

**Comment by Krealle** [comment]
is this happening without any changes to the raid roster, re-specs included?

**Comment by buzzblizard** [comment]
Correct. No new players joining or leaving the raid, just swapping back and forth between the default sort and RaidSortOptions

**Comment by Krealle** [comment]
I pushed a potential fix, let me know if it resolves the issue.

**Comment by buzzblizard** [comment]
14x Cell/Utils.lua:619: invalid value (table) at index 1 in table for 'concat'
[string "=[C]"]: in function `concat'
[string "@Cell/Utils.lua"]:619: in function `TableToString'
[string "-- More options for raid sorting

<< Further contents of RaidSortOptions.lua >>

This was during a quick LFR run.

**Comment by Krealle** [comment]
That has already been fixed 🫡

**Comment by buzzblizard** [comment]
Thanks so much for reacting quickly to this!

Raid roster is now sorted, with expected updates when new raid members join or leave. 

I've had the raid roster locked in the Cell default state if combat starts during the grace period before RaidSortOptions is triggered - I'm assuming this is expected and there's very little we can do about it.

**Comment by Krealle** [comment]
You can lower the grace period by setting QUE_TIMER lower, lowest possible is 0.5

**Comment by MarHuth** [comment]
I joined a Worldboss Raid and waited in Valdrakken to see how the Addon would sort things. Most of the time the sorting was correct. I couldnt really check if people from the raid switched spec which could explain some resorting but there were times when the addon would resort every 10-45 sec without roster changes. Sometimes my player frame would be number 1 sometimes not.

> [16:27:27] Hauntedmage-Ahn'Qiraj has left the raid group. 
> [16:27:28] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Gravieboom-Draenor[raid16], Spellfingers-Draenor[raid31], Brâdley-Ysondre[raid32], Berdant-Sanguino[raid33] 
> [16:27:56] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Spellfingers-Draenor[raid31], Berdant-Sanguino[raid33] 
> [16:28:41] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Berdant-Sanguino[raid33] 
> [16:28:58] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Berdant-Sanguino[raid33] 
> [16:29:37] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Berdant-Sanguino[raid33] 
> [16:29:42] Todesrasen-Frostwolf has left the raid group. 
> [16:29:43] [RaidSortOptions] Info: Unable to find spec info(2) for: Náirá-Blackrock[raid3], Sticious-TarrenMill[raid4], Bubbleman-Ragnaros[raid5], Skinirb-Sanguino[raid12], Berdant-Sanguino[raid32] 

this was the Info output in the chat and this is my Sorting_order:
local SORTING_ORDER = {
    "PLAYER",
    --"SPECROLE",
    --"SPEC",
    --"NAME",
    "ROLE"
}
Pretty sure that sometimes the addon defaults get re-applied because when i switched my current role around in the addon config that would switch where my Frame would end up if it is not the first.
